scripts/create-patch.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+echo "Creating patch..."
```

Comment:
In the beginning we should have IMO and in the other script file too:

`cd "$(dirname "$0")"` and then based on that `cd playwright` afterwards

Reply:
What does it do?

Reply:
$0 = the full path of the scripts/create-patch.sh 
dirname = get the dirname of it
cd -> navigate to it
-> you are able to execute the script from everywhere and you don't depend on the CWD. Thats what we usually do in PW upstream scripts.

Reply:
Okay not that good in bash :/